### DISCUSSION TITLE
Problems with GDL installer

### DISCUSSION THREAD

**Discussion by Gostguest** [gnudatalanguage/gdl#878] [Q&A]
Hi! I am having a problem istalling GDL on Windows 10. I have used MSYS2 for installation. But on the step Build GDL:$ ls install # compiled results will be here does not working. 'ls: cannot access 'install': No such file or directory'. I suppose that there are missing some files from https://github.com/gnudatalanguage/gdl. 
Thank you in advance for all your answers and suggestions.

**Comment by slayoo** [comment]
Please clarify which way of installing are you using?
Is it by using one of the artefacts from the GithubActions build? (e.g., https://github.com/gnudatalanguage/gdl/actions/runs/510150486) 
Thanks for posting.

**Comment by pjb7687** [comment]
I suppose you followed the steps in the `Compilation` section, may I ask you to post the whole build log? Unless it is difficult for me to reproduce the problems you have encountered. Thanks!

**Reply by Gostguest** [reply]
Thank you. I used MSYS2. Like it is described on the page https://github.com/gnudatalanguage/gdl/wiki/GDL-on-Windows

**Reply by pjb7687** [reply]
Thanks, but I mean you can copy and paste everything you have seen in the MSYS2 terminal... So that I can investigate what was the problem.

**Comment by Gostguest** [comment]
Thank all of you/ Finally, I have installed GDL, but i cannot find a debugger.
